fabric-network/types/index.d.ts

```diff
@@ -99,6 +99,7 @@ export interface TransientMap {
 }
 export class Transaction {
 	getName(): string;
+	getTransactionId(): string;
```

Comment:
Based on the code and tests, the return type should be `string | null`

Reply:
fixed